hw20/roles/hw202/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+#Copy to hosts
+- name: "Copy files to hosts"
+  become: true

```

Comment:
Do you need `root` to copy files to tmp?

Reply:
no. i fix that issue.